app/containers/message/Markdown.js

```diff
@@ -95,7 +115,7 @@ export default class Markdown extends React.Component {
 				style={{
 					paragraph: styles.paragraph,
 					text: {
-						fontSize: 16,
+						fontSize: isMessageContainsOnlyEmoji ? 50 : 16,
```

Comment:
We have two options here:

## First option:
WhatsApp does it slightly different and IMHO we should do something similar.
They show bigger emojis, but in degrees.
If it's only one emoji, it's very big (fontSize ~46).
If it's two emojis, it's fontSize ~36.
If it's three, fontSize is normal, 26.
Otherwise it's 16.
<img src="https://user-images.githubusercontent.com/804994/55727968-88453600-59e9-11e9-9e4f-50a66a8361f8.png" width="350" />

## Second option:
One bigger emoji style (like now), but limiting the length.
<img src="https://user-images.githubusercontent.com/804994/55728081-c3e00000-59e9-11e9-8946-b74685f92af9.png" width="350" />

Biggest emoji size should be 46.

I prefer the first one, but I'll leave it for you to choose.
What do you think?

Reply:
Note: when testing emoji lengths, don't forget one emoji length === 2 ;)

Reply:
Another option would be to make it like in skype:
  if 1 emoji - then big emoji
  if 2+ emoji - then small emoji